### PR TITLE
Docassembly prod config

### PR DESCRIPTION
**Pull request by linusnorton**

### CODE REVIEW

File: k8s/prod/common/dg/dg-docassembly.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    version: 0.2.4
+  values:
+    docassembly-api:
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/dg/docassembly:aat-c0681129
```
Comment (adusumillipraveen) [inline k8s/prod/common/dg/dg-docassembly.yaml]: aat ?

File: k8s/prod/common/dg/dg-docassembly.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      ingressHost: dg-docassembly-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline k8s/prod/common/dg/dg-docassembly.yaml]: aat ?

File: k8s/prod/common/dg/dg-docassembly.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
+    global:
+      environment: prod
+      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```
Comment (adusumillipraveen) [inline k8s/prod/common/dg/dg-docassembly.yaml]: this needs changing as well, look at other prod apps

**Comment by linusnorton** [comment]
I've updated the code owners and I'm just checking whether we need the DNS hack for docmosis then we should be good

**Comment by linusnorton** [comment]
@adusumillipraveen I believe this is good to go now

File: CODEOWNERS [hunk 4]
```diff
@@ -66,6 +66,7 @@ k8s/aat/common/financial-remedy/finrem-ns.yaml @hmcts/financial-remedy
 k8s/**/common/dg/namespace.yaml @hmcts/evidence
 k8s/**/common/dg/identity.yaml @hmcts/evidence
 k8s/aat/common/dg/dg-docassembly.yaml @hmcts/evidence
+k8s/prod/common/dg/dg-docassembly.yaml @hmcts/evidence
```
Comment (adusumillipraveen) [inline CODEOWNERS]: change it to k8s/**/common/dg/dg-docassembly.yaml @hmcts/evidence ?

**Pull request merged**